src/søknad/steg/4-barnasbosted/OmAndreForelder.tsx

```diff
@@ -138,16 +131,23 @@ const OmAndreForelder: React.FC<Props> = ({
       },
     };
 
+    let verdi = '';
     if (svar.id === EHvorforIkkeOppgi.donorbarn) {
-      delete forelder.ikkeOppgittAnnenForelderBegrunnelse;
+      verdi = 'Donor';
```

Comment:
Er dette riktig for engelsk søknad?

Reply:
Godt poeng! Vi hadde flaks med at det både er 'Donor' på norsk og engelsk